tst/EnergyPlus/unit/OASystemHWPreheatCoil.unit.cc

```diff
@@ -2020,6 +2022,11 @@ namespace EnergyPlus {
 		CpAir = PsyCpAirFnWTdb( Node( AirInletNodeNum ).HumRat, Node( AirInletNodeNum ).Temp );
 		EXPECT_NEAR( WaterCoil( 1 ).TotWaterHeatingCoilRate, WaterCoil( 1 ).InletAirMassFlowRate * CpAir * ( WaterCoil( 1 ).OutletAirTemp - WaterCoil( 1 ).InletAirTemp ), 1.0 ); 
 
+		// test that OA sys water coil bypasses normal controller calls before air loop simulation
+		EXPECT_EQ( "PREHEAT COIL CONTROLLER", HVACControllers::ControllerProps( 1 ).ControllerName );
+		EXPECT_TRUE( HVACControllers::ControllerProps( 1 ).BypassControllerCalc );
+		// test that water coil knows which controller controls the HW coil
+		EXPECT_EQ( WaterCoil( 1 ).ControllerIndex, 1 );
 	}
 
 }
```

Comment:
This is an existing unit test that works. So why does Git warn of the missing line at the end of a unit test?